x/poolx/keeper/keeper.go

```diff
@@ -0,0 +1,42 @@
+package keeper
+
+import (
+	"fmt"
+
+	"github.com/tendermint/tendermint/libs/log"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+	"github.com/cosmos/cosmos-sdk/x/poolx/types"
+)
+
+type Keeper struct {
+	cdc        codec.Codec
+	storeKey   sdk.StoreKey
+	memKey     sdk.StoreKey
+	paramSpace paramtypes.Subspace
+}
+
+func NewKeeper(
+	cdc codec.Codec,
+	storeKey sdk.StoreKey,
+	memKey sdk.StoreKey,
+	paramSpace paramtypes.Subspace,
+) *Keeper {
+	// set KeyTable if it has not already been set
+	if !paramSpace.HasKeyTable() {
+		paramSpace = paramSpace.WithKeyTable(types.ParamKeyTable())
+	}
```

Comment:
I copied this and don't fully understand why. Also doesn't seem to matter to the current test suite.  #cargoculted